document/core/valid/modules.rst

```diff
@@ -181,7 +181,7 @@ Element segments :math:`\elem` are classified by :ref:`segment types <syntax-seg
    }
 
 
-:math:`\{ \EINIT~e^\ast \}`
+:math:`\{ \ETYPE~et, \EINIT~e^\ast \}`
 ...........................
```

Comment:
Fix underline to make Sphinx happy

Reply:
Yeah, I noticed that later and fixed in 88cb63830ff539cfe7501dc1900c4ab968fe9395.